Makefile.include

```diff
@@ -566,6 +566,13 @@ term: $(filter flash, $(MAKECMDGOALS)) $(TERMDEPS)
 	$(call check_cmd,$(TERMPROG),Terminal program)
 	$(TERMPROG) $(TERMFLAGS)
 
+# Term without the pyterm added logging
+# TERMFLAGS must be exported for `jlink.sh term_rtt`.
+cleanterm: export PYTERMFLAGS += --noprefix
+cleanterm: $(filter flash, $(MAKECMDGOALS)) $(TERMDEPS)
+	$(call check_cmd,$(TERMPROG),Terminal program)
```

Comment:
should we keep on repeating bad patterns? I'm referring to checking the command.

Reply:
I still see it as a different task to remove the bad pattern that I did not want to mix with the other change here.